lib/Raven/ErrorHandler.php

```diff
@@ -92,12 +95,12 @@ public function handleException($e, $isError = false, $vars = null)
 
     public function handleError($code, $message, $file = '', $line = 0, $context=array())
     {
-        if ($this->error_types & $code & error_reporting()) {
+        if ($this->error_types & $code) {
             $e = new ErrorException($message, 0, $code, $file, $line);
             $this->handleException($e, true, $context);
         }
 
-        if ($this->call_existing_error_handler) {
+        if (error_reporting() & $code && $this->call_existing_error_handler) {
```

Comment:
looking at this, maybe we want to not regress here -- that is always call the existing handler if we were instructed to and let them deal with checking error_reporting